lightkurve/correctors/pldcorrector.py

```diff
@@ -85,52 +84,151 @@ class PLDCorrector(Corrector):
         (arXiv:1702.05488)
     .. [4] EVEREST pipeline webpage, https://rodluger.github.io/everest
     """
-    def __init__(self, tpf):
+
+    def __init__(self, tpf, aperture_mask=None):
+
         self.tpf = tpf
+        if aperture_mask is None:
+            aperture_mask = tpf.create_threshold_mask(2)
+        self.lc = self.tpf.to_lightcurve(aperture_mask=aperture_mask)
 
-        self.flux = tpf.flux
-        self.flux_err = tpf.flux_err
-        self.time = tpf.time.value
+        super(PLDCorrector, self).__init__(lc=self.lc)
 
-    def correct(self, aperture_mask=None, cadence_mask=None, gp_timescale=30,
-                use_gp=True, pld_order=2, n_pca_terms=10, pld_aperture_mask=None):
-        r"""Returns a PLD systematics-corrected LightCurve.
+    def __repr__(self):
+        return 'PLDCorrector (LC: {})'.format(self.lc.label)
+
+    @property
+    def X(self):
+        return self.dm
+
```

Comment:
I recommend removing the `X` property because it does not appear to be used in the code, and the `RegressionCorrector` parent class already exposes this property both via `self.dmc` and `self.design_matrix_collection`.  There should only be one way to access the design matrix!
```suggestion
```